src/homeworks/test_48/index.ts

```diff
@@ -0,0 +1,23 @@
+type dataInput = {
```

Comment:
I tipi dovrebbero cominciare con la lettera maiuscola.
```suggestion
type DataInput = {
```